### PR TITLE
fixing mistakes

### PR DESCRIPTION
**Pull request by mairaw**
While reviewing some of the CLI topics today, I've realized that working late on a Sunday night clouds your vision and makes your brain not work properly. 😄 

Fixing some mistakes for the new dotnet help command.

### CODE REVIEW

File: docs/core/tools/dotnet-help.md [hunk 1]
```diff
@@ -38,10 +38,6 @@ Prints out a short help for the command.
...
 
-List the project references for the specified project:
+Opens the documentation page for the [dotnet new](dotnet-new.md) command:
 
-`dotnet list app/app.csproj reference`
-
-List the project references for the project in the current directory:
-
-`dotnet list reference`
+`dotnet help new`
```
Comment (JRAlexander) [inline docs/core/tools/dotnet-help.md]: Should this example be wrapped in a console code sample markup to be consistent?

**Review by JRAlexander — CHANGES_REQUESTED** [review]
Looks great! Only one small item.

File: docs/core/tools/dotnet-help.md (hunk 1 above)
Comment (mairaw) [inline docs/core/tools/dotnet-help.md]: Actually, all of our CLI examples are inline code. So to be consistent, I'd have to change all others. We currently don't a language identifier for .NET Core CLI. I can make a test on a separate branch to see which one looks better.

Comment (JRAlexander) [inline docs/core/tools/dotnet-help.md]: Understand. LGTM then.

**Review by JRAlexander — APPROVED** [review]
LGTM

**Pull request merged**